main/amavisd-new/amavisd.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+
+pidfile="/run/amavisd.pid"
+
+name="amavisd"
+command="/usr/sbin/$name"
+command_args="-c /etc/amavisd.conf foreground"
+command_background="yes"
+
+start_stop_daemon_args="--user $USER:$GROUP"
```

Comment:
Please name variables in lowercase. Moreover `USER` has a bit special meaning and it may cause bad consequences.